### PR TITLE
Update IAM paginators

### LINKED ISSUE #919: Add paginators for 4 IAM actions
It looks like the following IAM actions manifest pagination
* `GetAccountAuthorizationDetails`
* `ListSshPublicKeys`
* `SimulatePrinciplePolicy`
* `SimulateCustomPolicy`

but don't have paginators in botocore

Would it be possible to add them to the [IAM paginators](https://github.com/boto/botocore/blob/develop/botocore/data/iam/2010-05-08/paginators-1.json)?

### PR DESCRIPTION
**Pull request by JordonPhillips**
Adds four new paginators:

* GetAccountAuthorizationDetails
* ListSSHPublicKeys
* SimulateCustomPolicy
* SimulatePrincipalPolicy

Closes #919

cc @kyleknap @jamesls

### CODE REVIEW

File: botocore/data/iam/2010-05-08/paginators-1.json [hunk 1]
```diff
@@ -58,108 +70,132 @@
...
     "ListRoles": {
       "input_token": "Marker",
-      "output_token": "Marker",
-      "more_results": "IsTruncated",
       "limit_key": "MaxItems",
+      "more_results": "IsTruncated",
+      "output_token": "Marker",
       "result_key": "Roles"
     },
+    "ListSAMLProviders": {
```
Comment (kyleknap) [inline botocore/data/iam/2010-05-08/paginators-1.json]: This is an invalid paginator. Because an input and output token are missing.

**Comment by JordonPhillips** [comment]
Removed the invalid paginator

**Comment by jamesls** [comment]
:shipit:

**Comment by JordonPhillips** [comment]
Added a change entry, after that I'll merge

**Pull request merged**